sbatch_scripts/gen_sbatch.py

```diff
@@ -354,64 +303,52 @@ def main():
         if args.sh_only:
             logger.info(f"Generated sbatch script: {script}")
             return
-        
-        # Make output folder for simulated data
+
         target.mkdir(exist_ok=True, parents=True)
         logger.info(f"Simulated data output folder: {target}")
 
         # Submits job
         submit_sbatch(script, logger)
-        logger.info(F"Job submitted successfully")
+        logger.info("Job submitted successfully")
 
     elif args.command == "jaxtrain":
```

Comment:
maybe we can use the occasion to add in "torchtrain" here as well? 
(since that's now include din LanFactory 0.5.1.